### DISCUSSION TITLE
how to add a sigmoid activation to the last layer of the monai.networks.nets.UNet architecture for deep learning applications？

### DISCUSSION THREAD

**Discussion by Guoxt** [Project-MONAI/MONAI#8039] [General]
monai.networks.nets.UNet(spatial_dims, in_channels, out_channels, channels, strides, kernel_size=3, up_kernel_size=3, num_res_units=0, act='PRELU', norm='INSTANCE', dropout=0.0, bias=True, adn_ordering='NDA')

how to add a sigmoid activation to the last layer of the monai.networks.nets.UNet architecture for deep learning applications？

**Comment by KumoLiu** [comment]
Hi @Guoxt, thanks for your interest here.
You can use `Activations` to apply the activation on the prediction.
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/post/array.py#L61

Such as:
```
post_trans = Compose([Activations(sigmoid=True), AsDiscrete(threshold=0.5)])

outputs = inference(val_inputs)
outputs = [post_trans(i) for i in decollate_batch(val_outputs)]
```

Hope it helps, thanks.
(move it to discussions for now, feel free to create another issue if meet further issue.)
